runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkTranslationContext.java

```diff
@@ -17,6 +17,7 @@
  */
 package org.apache.beam.runners.spark.translation;
 
+import com.sun.istack.Nullable;
```

Comment:
This is probably not the `Nullable` you want. CC @tysonjh since I found this from a failure on his PR.

Reply:
Correct, use javax nullable instead please.

I have local changes to replace the other occurrences of the istack one.